TimeTable/Categories/WorkTime/WorkTimeViewModel.swift

```diff
@@ -169,10 +169,7 @@ class WorkTimeViewModel: WorkTimeViewModelType {
     // MARK: - Private
     private func validateInputs() throws {
         guard .none != task.project else { throw UIError.cannotBeEmpty(.projectTextField) }
-        guard !task.body.isEmpty else { throw UIError.cannotBeEmpty(.taskTextField) }
-        if task.allowsTask && task.url == nil {
-            throw UIError.cannotBeEmpty(.taskURLTextField)
-        }
+        guard !task.body.isEmpty || (task.allowsTask && task.url != nil) else { throw UIError.cannotBeEmpty(.taskTextField) }
```

Comment:
Please change error that is thrown up. For e.g. something that will say that the user has to give task name or URL. Please remember that you have to update `Localizable.strings` file.